tests/test_modeling_flaubert.py

```diff
@@ -399,3 +399,19 @@ def test_model_from_pretrained(self):
         for model_name in FLAUBERT_PRETRAINED_MODEL_ARCHIVE_LIST[:1]:
             model = FlaubertModel.from_pretrained(model_name)
             self.assertIsNotNone(model)
+
+
+@require_torch
+class FlaubertModelIntegrationTest(unittest.TestCase):
+    @slow
+    def test_inference_no_head_absolute_embedding(self):
+        model = FlaubertModel.from_pretrained("xlm-mlm-en-2048")
```

Comment:
Could you leverage a Flaubert checkpoint? For example the following: https://huggingface.co/flaubert/flaubert_base_cased